packages/api/tests/serial/es/test-es-indexer.js

```diff
@@ -24,6 +24,31 @@ const granuleFailure = require('../../data/granule_failed.json');
 const pdrFailure = require('../../data/pdr_failure.json');
 const pdrSuccess = require('../../data/pdr_success.json');
 
+const granuleFileToRecord = (granuleFile) => {
+  const granuleRecord = {
+    fileSize: 12345,
+    ...granuleFile,
+    key: aws.parseS3Uri(granuleFile.filename).Key,
+    fileName: granuleFile.name,
+    checksum: granuleFile.checksumValue
+  };
+
+  if (granuleFile.path) {
+    granuleRecord.source = `https://07f1bfba.ngrok.io/granules/${granuleFile.name}`;
+  }
+
+  delete granuleRecord.checksumValue;
+  delete granuleRecord.filename;
+  delete granuleRecord.name;
+  delete granuleRecord.url_path;
+  delete granuleRecord.path;
+
+  if (isNil(granuleRecord.checksum)) delete granuleRecord.checksum;
+  if (isNil(granuleRecord.checksumType)) delete granuleRecord.checksumType;
+
```

Comment:
could you use the filterDatabaseProperties + removeNilProperties for same effect?

Reply:
I could in fact.  Fixed.